src/contributors/pervolarakis.json

```
@@ -0,0 +1,5 @@
+{
+  "name": "Mike Pervolarakis",
+  "desc": "Wannabe android developer",
```

```suggestion
  "desc": "Aspiring Developer",
```
Update `desc`, with something you're doing. If you're a student you can add `student` to it or else any designation you want.